src/ML-PACE/pair_grace.h

```diff
@@ -0,0 +1,93 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   LAMMPS development team: developers@lammps.org
+
+   This software is distributed under the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/*
+Copyright 2024 Yury Lysogorskiy^1,  Anton Bochkarev^1, Ralf Drautz^1
+
+^1: Ruhr-University Bochum, Bochum, Germany
+*/
+
+//
+// Created by Lysogorskiy Yury on 27.03.24
+//
+
+#ifndef NO_GRACE_TF
+#ifdef PAIR_CLASS
+// clang-format off
+PairStyle(grace,PairGRACE);
+// clang-format on
+#else
+
+#ifndef LMP_PAIR_GRACE_H
+#define LMP_PAIR_GRACE_H
+
+#include "pair.h"
+// CppFlow headers
```

Comment:
No implementation headers must be included in the header file. I don't see any use of any definitions in them, anyway.
Please follow the PIMPL programming pattern correctly.